Data/Sys/wiitdb-en.txt

```diff
@@ -0,0 +1,9305 @@
+GF5S69 = FIFA Football 2005
+GF6D69 = FIFA 06
+GF6E69 = FIFA Soccer 06
+GF6F69 = FIFA 06
+GF6H69 = FIFA 06
+GF6P69 = FIFA 06
+GF6S69 = FIFA 06
+GF7E01 = Star Fox: Assault
+GF7P01 = Star Fox Assault
+GF8E69 = FIFA Street
+GF8P69 = FIFA Street
+GFAD69 = FIFA Soccer 2003
+GFAE69 = FIFA Soccer 2003
+GFAF69 = FIFA Soccer 2003
+GFAI69 = FIFA Football 2003
+GFAJ69 = FIFA Soccer 2003
+GFAP69 = FIFA Football 2003
+GFAS69 = FIFA Football 2003
+GFBE5D = FireBlade
+GFBP5D = Fireblade
+GFCP69 = F1 Career Challenge
+GFDD69 = Freedom Fighters
+GFDE69 = Freedom Fighters
+GFDF69 = Freedom Fighters
+GFDP69 = Freedom Fighters
+GFEE01 = Fire Emblem: Path of Radiance
+GFEJ01 = Fire Emblem: Souen no Kiseki
+GFEP01 = Fire Emblem: Path of Radiance
+GFFE5D = Freaky Flyers
+GFGEA4 = Frogger Beyond
+GFGPA4 = Frogger Beyond
+GFHP6V = Neighbours From Hell
+GFIE69 = 2002 FIFA World Cup
+GFKE69 = Freekstyle
+GFKP69 = Freekstyle
+GFMJAF = Family Stadium 2003
+GFNJG2 = Finding Nemo
+GFOE78 = The Fairly OddParents - Shadow Showdown
+GFPEA4 = Frogger: Ancient Shadow
+GFQEA4 = Frogger's Adventures: The Rescue
+GFSD69 = 2002 FIFA World Cup
+GFSE69 = FIFA Soccer 2002
+GFSF69 = 2002 FIFA World Cup
+GFSJ69 = FIFA Soccer 2002
+GFSP69 = 2002 FIFA World Cup
+GFTE01 = Mario Golf: Toadstool Tour
+GFTJ01 = Mario Golf: Family Tour
+GFTP01 = Mario Golf: Toadstool Tour
+GFUE4Z = Future Tactics: The Uprising
+GFUP6V = Future Tactics: The Uprising
+GFVE5D = NFL Blitz Pro
+GFWE78 = The Fairly OddParents: Breakin' Da Rules
+GFXE5D = Freestyle Metal X
+GFYE69 = FIFA Street 2
+GFYP69 = FIFA Street 2
+GFZE01 = F-Zero GX
+GFZJ01 = F-Zero GX
+GFZJ8P = F-Zero AX
+GFZP01 = F-Zero GX
+GG1J08 = Donkey Kong: Jungle Fever
+GG2E4Z = Trigger Man
+GG3J08 = Donkey Kong: Banana Kingdom
+GG4E08 = Gotcha Force
+GG4J08 = Gotcha Force
+GG4P08 = Gotcha Force
+GG5E52 = Cabela's Big Game Hunter 2005
+GGAJB2 = Mobile Suit Gundam: The Ace Pilot
+GGCE0A = Goblin Commander: Unleash The Horde
+GGCP0A = Goblin Commander: Unleash the Horde
+GGEE41 = Beyond Good & Evil
+GGEP41 = Beyond Good And Evil
+GGEX41 = Beyond Good And Evil
+GGEY41 = Beyond Good And Evil
+GGFJ01 = GiFTPiA
+GGNE5D = The Grim Adventures of Billy and Mandy
+GGPE01 = Mario Kart Arcade GP
+GGPE02 = Mario Kart Arcade GP 2
+GGPJ02 = Mario Kart Arcade GP 2
+GGPJAF = SD Gundam Gashapon Wars
+GGPJB2 = SD Gundam Gashapon Wars
+GGRD41 = Tom Clancy's Ghost Recon
+GGRE41 = Tom Clancy's Ghost Recon
+GGRP41 = Tom Clancy's Ghost Recon
+GGSEA4 = Metal Gear Solid: The Twin Snakes
+GGSJA4 = Metal Gear Solid: The Twin Snakes
+GGSPA4 = Metal Gear Solid: The Twin Snakes
+GGTE01 = Chibi-Robo!
+GGTJ01 = Chibi-Robo!
+GGTP01 = Chibi-Robo!
+GGVD78 = Spongebob Squarepants : The Movie
+GGVE78 = SpongeBob SquarePants: The Movie
+GGVP78 = Spongebob Squarepants: The Movie
+GGVX78 = Spongebob Squarepants : The Movie
+GGYE41 = Tom Clancy's Ghost Recon 2
+GGYP41 = Tom Clancy's Ghost Recon 2
+GGZE52 = Madagascar
+GGZH52 = Madagascar
+GGZI52 = Madagascar
+GGZJB2 = Madagascar
+GGZP52 = M
```

Comment:
This line is normally at the top, but we need the whole file to be sorted for `comm` to work. I don't think this is an issue though, since it is trivial to find this line with something like grep.